### PR TITLE
Parse Snowflake USE ROLE and USE SECONDARY ROLES

### PR DESCRIPTION
**Pull request by yoavcloud**
This PR adds support for Snowflake `USE ROLE <role>` and `USE SECONDARY ROLES` as described here: https://docs.snowflake.com/en/sql-reference/sql/use-secondary-roles

### CODE REVIEW

**Review by alamb — APPROVED** [review]
Makes sense to me -- thank you @yoavcloud 

FYI @iffyio

File: tests/sqlparser_snowflake.rs [hunk 1]
```diff
@@ -2709,6 +2709,20 @@ fn parse_use() {
...
             ])))
         );
+        std::assert_eq!(
+            snowflake().verified_stmt(&format!("USE ROLE {0}my_role{0}", quote)),
+            Statement::Use(Use::Role(ObjectName(vec![Ident::with_quote(
+                quote,
+                "my_role".to_string(),
+            )])))
+        );
+        std::assert_eq!(
```
Comment (iffyio) [inline tests/sqlparser_snowflake.rs:2719]:
```suggestion
        assert_eq!(
            snowflake().verified_stmt(&format!("USE ROLE {0}my_role{0}", quote)),
            Statement::Use(Use::Role(ObjectName(vec![Ident::with_quote(
                quote,
                "my_role".to_string(),
            )])))
        );
        assert_eq!(
```
if these were autoimported?

**Review by iffyio — COMMENTED** [review]
Thanks @yoavcloud! Just one minor comment on the std assert_eq import

File: tests/sqlparser_snowflake.rs (hunk 1 above)
Comment (yoavcloud) [reply tests/sqlparser_snowflake.rs]: Fixed across the board

**Review by iffyio — APPROVED** [review]
LGTM! Thanks @yoavcloud!

**Pull request merged**